src/test/integration/org/apache/cassandra/sidecar/testing/CassandraSidecarTestContext.java

```diff
@@ -142,6 +182,8 @@ public String toString()
     public void close()
     {
         instancesConfig.instances().forEach(instance -> instance.delegate().close());
+        // TODO: Do we need to close sessions?
+//        sessionProviders.
```

Comment:
It is to be resolved?

Reply:
Yup - thanks for catching it... was a simple one-liner I just hadn't remember to get back to.

Reply:
To be a bit more specific, I'm fairly certain that shutting down the Sidecar server itself will eventually close these, but in case somehow the sidecar server doesn't, it makes sense to close them explicitly here.